exercises/java/src/main/java/com/adaptionsoft/games/uglytrivia/Category.java

```diff
@@ -0,0 +1,28 @@
+package com.adaptionsoft.games.uglytrivia;
+
+import java.util.LinkedList;
+
+public class Category {
```

Comment:
Good use of another class to free up the Game class.

Reply:
Thank you